Compliance error is on line 65, which I do not touched.
Do I have to commit the clang fix ?

```
Run if [[ ! -s "compliance.xml" ]]; then
Error: See https://docs.zephyrproject.org/latest/contribute/guidelines.html#coding-style for more details.

 LONG_LINE: line length of 104 exceeds 100 columns
File:drivers/sensor/asahi_kasei/akm09918c/akm09918c_async.c
Line:65
Compliance error, check for error messages in the "Run Compliance Tests" step
You can run this step locally with the ./scripts/ci/check_compliance.py script.
Error: Process completed with exit code 1.
```

--- a/drivers/sensor/asahi_kasei/akm09918c/akm09918c_async.c
+++ b/drivers/sensor/asahi_kasei/akm09918c/akm09918c_async.c
@@ -60,10 +60,10 @@ void akm09918c_submit(const struct device *dev, struct rtio_iodev_sqe *iodev_sqe
 		data->rtio_ctx, data->iodev, AKM09918C_REG_CNTL2, AKM09918C_CNTL2_SINGLE_MEASURE);
 	struct rtio_sqe *cb_sqe = rtio_sqe_acquire(data->rtio_ctx);
 
-	writeByte_sqe->flags |= RTIO_SQE_CHAINED;
-	rtio_sqe_prep_callback_no_cqe(cb_sqe, akm09918_after_start_cb, (void *)iodev_sqe, NULL);
-
 	if (writeByte_sqe != NULL && cb_sqe != NULL) {
+		writeByte_sqe->flags |= RTIO_SQE_CHAINED;
+		rtio_sqe_prep_callback_no_cqe(cb_sqe, akm09918_after_start_cb, (void *)iodev_sqe,
+					      NULL);
 		rtio_submit(data->rtio_ctx, 0);
 	} else {
 		rtio_sqe_drop_all(data->rtio_ctx);

